src/microbe_stage/ISpawned.cs

```diff
@@ -8,4 +8,9 @@ public interface ISpawned : IEntity
     ///   greater than this, it is despawned.
     /// </summary>
     int DespawnRadiusSquared { get; set; }
+
+    /// <summary>
+    ///   How much this entity contributes to the entity limit (based on Node complexity)
+    /// </summary>
+    int EntityWeight { get; }
```

Comment:
I have a feeling that this should really be a float because I imagine the performance impact of some entities related to other ones might be something like 30-50% instead of 100% or 200% and so on.

Reply:
How much of the existing entity counting framework should be adjusted to account for this? Entity count is an int everywhere it turns up, so this would mean either rounding these float values (which to me seems redundant) or having a float total entity count.

Reply:
I'd change the entity count in just the spawn system to store the value as a float, then it can still be compared against the limits defined as int just fine.